packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -3184,6 +3184,67 @@ addEventListener('fetch', event => {});`
     });
   });
 
+  describe("[define]", () => {
+    it("should be able to define values that will be substituted", async () => {
+      writeWranglerToml({
+        main: "index.js",
+        define: {
+          abc: "123",
```

Comment:
What happens if this expression is not valid JS syntax? Will esbuild break or just substitute the rubbish?